package.json

```diff
@@ -23,16 +23,18 @@
     "lib": "./lib/memjs"
   },
   "scripts": {
-    "test": "eslint ./lib/memjs/ ./test/ && tap -R spec ./test/*.js",
+    "test": "eslint ./lib/memjs/ ./test/ && tap  --no-coverage -R spec ./test/*.js",
```

Comment:
why `--no-coverage`? just curious.

Reply:
because the ancient version of tap they had been using didn't print coverage and I wanted to preserve that behavior.

also, because the tap coverage report takes up basically a full screen and pushes out the output from running the tests that one really cares about - the status of the tests.